mobify/pinny
# Fix 69: iOS 7 shifting focus points due to address bar showing

We found a way to fix the shifted focus points when it happened. We need to do focus and then blur on an input element. However, on iOS 7.x, the resize event does not fire when the address bar/footer shows or hides.

Reference: https://steveush.wordpress.com/2014/03/13/safari-on-ios7-address-and-bookmark-bars-fixed-position-bug/

We are thinking to attempt the solution above but only poll the window height while pinny is open. https://gist.github.com/scalvert/1498b1e8a4e5ca89df17
 
Status: **Opened for visibility**

Reviewers: @scalvert 
JIRA: https://github.com/mobify/pinny/issues/69

How to test drive this PR:
1. Use a device running iOS 7.0.x
2. `grunt serve`
3. Navigate to the examples 
4. Partially scroll the page
5. Open a pinny
6. Tap the bottom ~50px to trigger the bottom status bar
7. Ensure pinny closes, and/or input fields have their cursor correctly positioned
